packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -279,12 +362,107 @@ export default class StatsAnalyzer extends EventsScope {
+      this.statsResults[mediaType][sendrecvType].headerBytesReceived = result.headerBytesReceived;
+      this.statsResults[mediaType][sendrecvType].jitter = result.jitter;
     }
   }
 
+  /**
+   * Processes InboundRTP stats result and stores
+   * @private
+   * @param {*} result
+   * @param {*} type
+   * @param {boolean} isSender
+   *
+   * @returns {void}
+   */
+  remotecandidate = (result, type, isSender) => {
+    if (!result) {
+      return;
+    }
+    const RemoteCandidateType = {};
+    const RemoteTransport = {};
+    const RemoteIpAddress = {};
+    const RemoteNetworkType = {};
+
+    if (!result.id) return;
+
+    const sendRecvType = isSender ? 'send' : 'recv';
+
+    if (!RemoteCandidateType[result.id]) {
+      RemoteCandidateType[result.id] = [];
+    }
+
+    if (!RemoteTransport[result.id]) {
+      RemoteTransport[result.id] = [];
+    }
+
+    if (!RemoteIpAddress[result.id]) {
+      RemoteIpAddress[result.id] = [];
+    }
+    if (!RemoteNetworkType[result.id]) {
+      RemoteNetworkType[result.id] = [];
+    }
+
+    if (result.candidateType && RemoteCandidateType[result.id].indexOf(result.candidateType) === 0) {
+      RemoteCandidateType[result.id].push(result.candidateType);
+    }
+
+    if (result.transport && RemoteTransport[result.id].indexOf(result.transport) === 0) {
+      RemoteTransport[result.id].push(result.transport);
+    }
+
+    if (result.ipAddress && RemoteIpAddress[result.id].indexOf(`${result.ipAddress}:${result.portNumber}`) === 0) {
+      RemoteIpAddress[result.id].push(`${result.ipAddress}:${result.portNumber}`);
+    }
+
+    if (result.networkType && RemoteNetworkType[result.id].indexOf(result.networkType) === 0) {
+      RemoteNetworkType[result.id].push(result.networkType);
+    }
+
+    this.statsResults.internal.candidates[result.id] = {
+      candidateType: RemoteCandidateType[result.id],
+      ipAddress: RemoteIpAddress[result.id],
+      portNumber: result.portNumber,
+      networkType: RemoteNetworkType[result.id],
+      priority: result.priority,
+      transport: RemoteTransport[result.id],
+      timestamp: result.timestamp,
+      id: result.id,
+      type: result.type
+    };
+
+    this.statsResults.connectionType.remote.candidateType = RemoteCandidateType[result.id];
+    this.statsResults.connectionType.remote.ipAddress = RemoteIpAddress[result.id];
+    this.statsResults.connectionType.remote.networkType = RemoteNetworkType[result.id];
+    this.statsResults.connectionType.remote.transport = RemoteTransport[result.id];
+
+    this.statsResults[type][sendRecvType].totalRoundTripTime = result.totalRoundTripTime;
+    this.statsResults[type][sendRecvType].bytesReceived = result.bytesReceived;
+    this.statsResults[type][sendRecvType].bytesSent = result.bytesSent;
+    this.statsResults[type][sendRecvType].packetsReceived = result.packetsReceived || undefined;
+    this.statsResults[type][sendRecvType].packetsSent = result.packetsSent || undefined;
```

Comment:
So if zero packets were received or sent (i.e. result.packetsReceived === 0), you would rather send `undefined` instead of zero?

Reply:
We cannoti did not see it